src/Message/Payload/Builder/MessagePayloadBuilderInterface.php

```diff
@@ -33,7 +33,10 @@ public function withClaims(array $claims): MessagePayloadBuilderInterface;
 
     public function withClaim($claim, $claimValue = null): MessagePayloadBuilderInterface;
 
-    public function withMessagePayloadClaims(MessagePayloadInterface $payload): MessagePayloadBuilderInterface;
+    public function withMessagePayloadClaims(
+        MessagePayloadInterface $payload,
+        array $exclusions = []
```

Comment:
I would consider this as a breaking change.

Reply:
Indeed, by practice you're right even if I doubt anyone override this low level interface of the lib.
By precaution, I'll apply a major semver bump, but will stale this PR to add more legit features for a v4.x (like PHP8 support for example)

Reply:
That should still be mentioned in the changelog as a breaking change, though, IMO.

Reply:
i'm preparing a new major release (for PHP8 especially), where this will be documented as BC